docs/en/seatunnel-engine/separated-cluster-deployment.md

```diff
@@ -382,6 +284,106 @@ TCP is the way we recommend to use in a standalone SeaTunnel Engine cluster.
 
 On the other hand, Hazelcast provides some other service discovery methods. For details, please refer to [hazelcast network](https://docs.hazelcast.com/imdg/4.1/clusters/setting-up-clusters).
 
+### 5.3 Persistence Configuration of IMap (This parameter is invalid on the Worker node)
```

Comment:
The section 5 named `Configuring SeaTunnel Engine Network Services`. So I think we should not put Imap Configuration in here. It is a part of `Configure SeaTunnel Engine
 `